schema-based-on-diagram.sql

```diff
@@ -0,0 +1,41 @@
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+  id              INT  PRIMARY KET GENERATED ALWAYS AS IDENTITY,
+  name            CHAR(100) NOT NULL, 
+  date_of_birth   DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id              INT  PRIMARY KET GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] There is a typo error on line 10. Kindly correct it from `KET` to `KEY`.

- [ ] Kindly ensure that a "join table" for the many-to-many relationships between medical_histories and treatments is created, even if it is not shown on the diagram.

- [ ] Kindly ensure that all foreign keys have indexes created. This will allow users to quickly retrieve records from your database file. [Here](https://www.codecademy.com/article/sql-indexes) is a good source to learn how to create indexes.